iGCS/Models/Craft/GCSCraftNotifications.m

```diff
@@ -24,4 +25,18 @@ + (void)didNavModeChangeFromLastHeartbeat:(mavlink_heartbeat_t) lastHeartbeat
     });
 }
 
++ (void)didMavModeChangeFromLastHeartbeat:(mavlink_heartbeat_t) lastHeartbeat
+                          andNewHeartbeat:(mavlink_heartbeat_t) newHeartbeat {
+
+    if (lastHeartbeat.base_mode != 0 &&
+        (lastHeartbeat.base_mode & MAV_MODE_FLAG_SAFETY_ARMED) == (newHeartbeat.base_mode & MAV_MODE_FLAG_SAFETY_ARMED)) {
+        return;
+    }
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        [[NSNotificationCenter defaultCenter]postNotificationName:GCSCraftNotificationsCraftMavModeDidChange
```

Comment:
Sorry, yes - I double confused myself. Renaming the notifications and associated methods will clear this up.

GCSCraftNotificationsCraftDidArmedStatusChange
craftArmedStatusDidChange
etc